### ISSUE TITLE
output-validate doesnt support non-root file paths

### ISSUE THREAD

**Issue by tsirlucas** [awslabs/video-on-demand-on-aws#61]
### How to reproduce
If you're using the `Source Metadata Option` triggers and passing a bucket `folder` on `destBucket`.

Ex: 
```
  {
    srcVideo: any,
    destBucket: `${environment.assetsBucket}/folder1/folder2`
  };
```

### Reason
I believe the reason is [the buildUrl](https://github.com/awslabs/video-on-demand-on-aws/blob/0337723738c05d2dfe6f9e0a85f529c362573eff/source/output-validate/index.js#L18) function, which is ignoring everything after the 3rd item when calling splice. In the past example, all urls would be incomplete because we would have more items.

### Test case
I have create [this fork](https://github.com/picter/video-on-demand-on-aws/commit/5d925a2103e960f144c5161b17e8dd067c4dd4a7) with a test case and a fix for the problem. I would gladly create a PR. Just need to confirm if there is any reason why the `buildUrl` function is ignoring everything after the 3rd item.

**Comment by dscpinheiro** [comment]
Hi @tsirlucas,

Thanks for the contribution!
If you create a PR, we'll look at including it in the next release of the solution.

**Comment by tsirlucas** [comment]
For some reason I'm receiving a 500 when trying to create the PR. I'm actually receiving a 500 just by accessing `https://github.com/awslabs/video-on-demand-on-aws/compare`. I'll try again in a few hours.

**Comment by tomnight** [comment]
This issue was fixed in the latest release v5.1.0
